packages/react/jest.config.js

```diff
@@ -101,9 +101,7 @@ module.exports = {
     './src/components/Page/PageTitle.jsx': { branches: 75 },
     './src/components/ImageCard/ImageCard.jsx': { branches: 71 },
     './src/components/TimeSeriesCard/TimeSeriesCard.jsx': {
-      statements: 77,
-      functions: 70,
-      lines: 77,
+      functions: 75,
```

Comment:
We are still leaving an exception here?

Reply:
We should increase in order to remove all exceptions from this file

Reply:
@davidicus The exceptions are all related to state dependant hover functions like [this one](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/5711c716475fe2c33b441450af0d9a8fad0e6934/packages/react/src/components/TimeSeriesCard/TimeSeriesCard.jsx#L361), or [ref based useEffects](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/5711c716475fe2c33b441450af0d9a8fad0e6934/packages/react/src/components/TimeSeriesCard/TimeSeriesCard.jsx#L414) that require the actual chart to be loaded and not just mocked. I wasn't sure what level of refactoring we wanted to employ. I can extract those hover functions, pass the `alertRanges` in, and return the hover function. That would extract those enough to be testable, and, I think, put us over the mark.

Reply:
@davidicus Extracted those methods and tested their outputs. That raises our coverage high enough to remove the exception without worrying about not mocking the chart.